lib/mpl_toolkits/axes_grid1/anchored_artists.py

```diff
@@ -53,6 +53,7 @@ def __init__(self, width, height, xdescent, ydescent,
 
         frameon : bool, default: True
             If True, draw a box around this artists.
+            If False, possibly allow image with proper transparency.
```

Comment:
Thanks for the PR,  OTOH, I had to read the original issue to have any idea what this meant.  Suggest

```
   If True, add a frame and solid background for the artist.  The frame's appearance can be changed with the `facecolor` and `edgecolor` kwargs.  
```

Though I admit I did not check that those kwargs actually work.  But presumably there is some kwarg that will set those properties.

Reply:
It looks like in this case we don't pass the kwargs to the right place to make that happen.

I agree that this wording is not clear, I agree with @jklymak that changing it to `"...dd a frame and solid background for the artist...."` is probably the right path.  It looks like if you access the `obj.patch` attribute you can change the color, visibility, ... after the fact.